docs/articles/faq/README.md

```diff
@@ -283,3 +283,17 @@ According to users' feedback, the following CI systems work best with TestCafe:
 * [TravisCI](../documentation/guides/continuous-integration/travis.md)
 * [CircleCI](../documentation/guides/continuous-integration/circleci.md)
 * [AppVeyor](../documentation/guides/continuous-integration/appveyor.md)
+
+### 'The browser can't open the page: can't establish a secure connection to the server'
+
+If your computer is connected to multiple networks (for instance, uses a VPN connection), TestCafe may host its internal proxy server on an IP that cannot be routed correctly due to VPN. In this case, the following error occurs:
```

Comment:
`TestCafe may host its internal proxy server on an IP` - it's too complicated.

Can be simplified as `TestCafe can calculate an host IP that cannot be reached/accesses...`

Reply:
Perhaps

> If your computer is connected to multiple networks (for instance, uses a VPN connection), TestCafe may pick an IP that is inaccessible due to VPN.

But then it's unclear ('pick for what?', 'unaccessible from where?'). 

@VasilyStrelyaev WDYT?

Reply:
Maybe: 

> If your computer is connected to multiple networks (for instance, if it uses VPN), TestCafe may identify the host with an IP that does not resolve to `localhost` due to VPN or routing issues.

Reply:
No.

The problem - By default, TestCafe uses the machine IP as host for the proxied URLs. When you connect via VPN, you IP can be calculated as IP of machine FROM you come. 
So, the solution - prevent using IP and use the 'localhost' alias.

Reply:
Oh, I see, I thought the problem was with reaching the correct IP...

Okay, cool, thanks, so:

> If your computer is connected to multiple networks (for instance, if it uses a VPN connection), TestCafe may detect the host IP incorrectly and thus generate testing links that do not open.

Reply:
> and thus generate testing links that do not open

It's not clear. Which testing links?

What about such variant?
`and cannot open the testing pages`.

Reply:
```suggestion
If your computer is connected to multiple networks (for instance, if it uses a VPN connection), TestCafe may be unable to open the testing pages.
```